File: src/main/java/mizuki/audience/CreativeManager.java
package mizuki.audience;

import redis.clients.jedis.Jedis;

class CreativeManager {

    private static Jedis jedis;

    CreativeManager(Jedis _jedis) {
        jedis = _jedis;
    }

    String getFillerCreativeByCategory(String category) {
        try {
            if (category == null || category.equals("")) {
                category = "filler";
            }

            if (jedis.exists(category)) {
                return jedis.get(category);
            }

            throw new RuntimeException("does not creative: " + category);
        } catch (Exception e) {
            e.printStackTrace();
        } finally {
            return jedis.get("filler");
        }
    }

}

File: src/main/java/mizuki/audience/CreativeManager.kt
package mizuki.audience

import redis.clients.jedis.Jedis
import redis.clients.jedis.Tuple

object CreativeManager {
    private var Jedis: Jedis = RedisManager.getJedisResource()

    fun getFillerCreativeByCategory(category: String): Creative {
        try {

            if (Jedis.exists(category)!!) {

                return Jedis.get(category)
            }

            throw RuntimeException("does not creative: " + category)
        } catch (e: Exception) {
            e.printStackTrace()
        } finally {
             val filler: Tuple = Jedis.zrangeWithScores("filler", 0L, 1L)
            filler

        }
    }
}
